File: Components/ASP.NET/CS/ASPxGridViewDemos/UserControls/ShowQueryBuilderButtonControl.ascx.cs
using System;
using System.Web.UI;

public partial class UserControls_ShowQueryBuilderButtonControl : UserControl {
    public UserControls_ShowQueryBuilderButtonControl(): base() {
        this.Init += showQueryBuilderButton_Init;
    }
    protected void showQueryBuilderButton_Init(object sender, EventArgs e) {
        this.hfShowTooltip["showTooltip"] = QueryBuilderHelper.NeedToShowTooltip(Session);
    }
}

File: Components/ASP.NET/VB/ASPxGridViewDemos/UserControls/ShowQueryBuilderButtonControl.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Web.UI

Partial Public Class UserControls_ShowQueryBuilderButtonControl
	Inherits UserControl
	Public Sub New()
		MyBase.New()
		AddHandler Me.Init, AddressOf showQueryBuilderButton_Init
	End Sub
	Protected Sub showQueryBuilderButton_Init(ByVal sender As Object, ByVal e As EventArgs)
		Me.hfShowTooltip("showTooltip") = QueryBuilderHelper.NeedToShowTooltip(Session)
	End Sub
End Class
